### PR TITLE
automation: Fix broken custom params verification

### PR DESCRIPTION
**Pull request by ricekot**
Do not break existing automation jobs that have not yet implemented
the new verifyCustomParameter method (AjaxSpiderJob, OpenApiJob, SoapJob, etc.)

Signed-off-by: ricekot <ricekot@gmail.com>

### CODE REVIEW

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
The commit message could also be tweaked.

**Review by psiinon — APPROVED** [review]

File: addOns/automation/src/test/java/org/zaproxy/addon/automation/AutomationJobUnitTest.java [hunk 1]
```diff
@@ -263,49 +263,75 @@ void shouldIgnoreNullParamValue() {
...
+        assertThat(progress.hasErrors(), is(equalTo(false)));
+        assertThat(progress.hasWarnings(), is(equalTo(false)));
+    }
 
-            // Then
-            assertThat(progress.hasErrors(), is(equalTo(false)));
-            assertThat(progress.hasWarnings(), is(equalTo(false)));
-        }
+    @Test
+    void shouldNotWarnForAddOnsWithDefaultVerifyCustomParameterImplementation() {
```
Comment (thc202) [inline addOns/automation/src/test/java/org/zaproxy/addon/automation/AutomationJobUnitTest.java:312]: `AddOns` → `Jobs`

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Comment by psiinon** [comment]
Still looks good to me :)

**Pull request merged**